docs/GettingStarted.md

```diff
@@ -1007,7 +1009,7 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 |  2.0          |  3.0 - 3.2               |
 |  2.1          |  3.0 - 4.2               |
 |  2.2 - 2.3    |  3.0 - 5.2               |
-|  2.4 - 2.5    |  4.2.8 - 5.2             |
+|  2.4 - 2.6    |  5.0 - 5.2               |
```

Comment:
The only reason we're raising the minimum version of Rails from 4.2.8 to 5.0 here is because we aren't testing Rails 4 with 2.4+ yeah?

If this is the case, I think we might want to leave it as it were, because although I like the idea that we only list the versions we actually test, Ruby 2.4 should actually work with Rails 4.2.8, and we don't want users getting the impression that the latest versions of Ruby don't work with Rails 4.

Reply:
Addressed.